### DISCUSSION TITLE
Dependency Injection in AM

### DISCUSSION THREAD

**Discussion by nmase88** [LuckyPennySoftware/AutoMapper#4363] [NA]
are you saying that you no longer support dependency injection in AM? 
Do you plan on updating the documentation to remove references to a deprecated package (AutoMapper.Extensions.Microsoft.DependencyInjection)?
Do you plan on updating nuget to mark said package as deprecated?

**Comment by jbogard** [comment]
are you saying that you no longer support dependency injection in AM? **No**
Do you plan on updating the documentation to remove references to a deprecated package (AutoMapper.Extensions.Microsoft.DependencyInjection)? **Sure**
Do you plan on updating nuget to mark said package as deprecated? **Sure**

If you are unhappy with service, you can apply for a refund

**Reply by jbogard** [reply]
The DI extension is folded in to the main AutoMapper library, but this hasn't been released yet. So we haven't written docs or upgrade guides or anything else that you've mentioned. Nothing was changed with that functionality, it just moved.
